hw/xbox/nv2a/nv2a_vsh.c

```diff
@@ -149,17 +147,17 @@ typedef struct VshOpcodeParams {
     bool C;
 } VshOpcodeParams;
 
-static const VshOpcodeParams ilu_opcode_params[] = {
-    /* ILU OP       ParamA ParamB ParamC */
-    /* ILU_NOP */ { false, false, false }, // Dxbx note : Unused
-    /* ILU_MOV */ { false, false, true  },
-    /* ILU_RCP */ { false, false, true  },
-    /* ILU_RCC */ { false, false, true  },
-    /* ILU_RSQ */ { false, false, true  },
-    /* ILU_EXP */ { false, false, true  },
-    /* ILU_LOG */ { false, false, true  },
-    /* ILU_LIT */ { false, false, true  },
-};
+// static const VshOpcodeParams ilu_opcode_params[] = {
+//     /* ILU OP       ParamA ParamB ParamC */
+//     /* ILU_NOP */ { false, false, false }, // Dxbx note : Unused
+//     /* ILU_MOV */ { false, false, true  },
+//     /* ILU_RCP */ { false, false, true  },
+//     /* ILU_RCC */ { false, false, true  },
+//     /* ILU_RSQ */ { false, false, true  },
+//     /* ILU_EXP */ { false, false, true  },
+//     /* ILU_LOG */ { false, false, true  },
+//     /* ILU_LIT */ { false, false, true  },
+// };
```

Comment:
`#if 0` or complete removal instead?

Why was this removed (and in such a weird way?)

Reply:
This was unused by the code entirely, but I left it in for now. Vote to remove entirely if it's not actually needed (but I'm not entirely sure tbh).